RocketChat/Rocket.Chat.ReactNative
# [FIX] Display emojis when used just after the text without any space

## Proposed changes

The emojis used just after the text without any space are not rendered as emojis.

## Issue(s)	


## How to test or reproduce

Send a message with some text and use an emoji just after the text like `Text:smiley:`  You will see that the emoji will not be rendered .

## Screenshots

## Types of changes



Before:
![emoji-before](https://user-images.githubusercontent.com/77742477/150422280-bc684609-c09f-4877-9044-1adda553a31b.png)


After:
![emoji-after](https://user-images.githubusercontent.com/77742477/150422042-8919cfe0-ecd5-4d97-8648-34d0e2e35b56.png)



## Checklist



## Further comments
